R/doeResponseSurfaceMethodology.R

```diff
@@ -311,8 +310,8 @@ doeResponseSurfaceMethodology <- function(jaspResults, dataset, options, ...){
   #Error 1: Does
 }
 
-.responseSurfaceContourPlot <- function(jaspResults, dataset, options, ready) {
-  contourPlot <- createJaspContainer(title = "Contour Plot")
+.responseSurfaceContourPlot <- function(jaspResults, dataset, options, data, lol) {
```

Comment:
can you try to use informative variable names? So instead of `lol` and `you_2`, something like aLongAndDescriptiveName (long names are fine)!